spring-cloud-deployer-sample-stream/src/main/java/sample/TickTock.java

```diff
@@ -71,8 +63,16 @@ private static AppDeploymentRequest createAppDeploymentRequest(String app, Strin
 			properties.put("spring.cloud.stream.bindings.input.group", "default");
 		}
 		AppDefinition definition = new AppDefinition(app, properties);
-		Map<String, String> environmentProperties = Collections.singletonMap(AppDeployer.GROUP_PROPERTY_KEY, stream);
-		AppDeploymentRequest request = new AppDeploymentRequest(definition, resource, environmentProperties);
+		Map<String, String> deploymentProperties = new HashMap<>();
+		deploymentProperties.put(AppDeployer.GROUP_PROPERTY_KEY, stream);
+		/*
+		 * This will allow output to be logged to the output of the process that started
+		 * the application.
+		 * Could be set per the entire deployment (i.e., 'deployer.inheritLogging=true') or per
```

Comment:
That's exactly what I meant ;). Let me know when you done with review and I'll address it with the last push.